.github/workflows/unit-tests.yaml

```diff
@@ -59,6 +59,13 @@ jobs:
       with:
         paths: 'junit-reports/report.xml'
 
+    - name: Report junit failures in jira
+      if: always() && github.event_name == 'push'
```

Comment:
Do we really want to run it on PR branches too? I'm afraid that would spam Jira with spurious reports from tests still under development.

Reply:
I've alread did that https://github.com/stackrox/stackrox/actions/runs/4699075635/jobs/8332123252#step:9:36

Reply:
Thats in dry-run mode, but maybe it's good enough...